build/patch/pre-autogen/libevent/common/000.patch

```diff
@@ -111,10 +82,10 @@ index 284341c..5b8b0a6 100644
 + */
 +#define _EVENT_HAVE_ARC4RANDOM_ADDRANDOM 1
```

Comment:
It *may* be possible that we should update this part of the diff to work differently. It seems now libevent does not use anymore leading underscored for variables it defines. Also, I wonder why forcing the check here when we also perform the check in the configure. I guess I'll need to double check that.